packages/commonwealth/package.json

```diff
@@ -218,7 +218,11 @@
     "web3-utils": "1.8.2",
     "yargs": "^17.7.2",
     "zod": "^3.22.4",
-    "zustand": "^4.3.8"
+    "zustand": "^4.3.8",
+    "@hicommonwealth/core": "0.0.0",
```

Comment:
not sure this is needed, I think yarn workspaces will hoist these automatically to the root